styles/globals.css

```diff
@@ -3,3 +3,8 @@
 @tailwind utilities;
 
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+ul {
+  list-style-type: disc !important;
```

Comment:
I think it's better to use tailwind `list-disc` class, instead of marking it as `!important` globally.

Because there is a possibility that we need to style a list differently

Reply:
change to tailwind list-disc